packages/documentation/docs/controls/_textarea-field_styleguide.mdx

```diff
@@ -0,0 +1,62 @@
+The Textarea component allows users to input multi-line text. It can be used in various forms where longer text input is required.
+
+![Textarea overview](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3814-1128&t=DtCmoFcLwhf7ke3S-4)
+1. Label
+2. Required field indicator (optional)
+3. Placeholder text
+4. Container
+5. Resize handle
+6. Helper or feedback text
+7. Counter
+
+## Options
+
+- **Label**: [See form field](./forms/forms-field.md)
+- **Value**: [See form field](./forms/forms-field.md)
+- **Required**: [See form field](./forms/forms-field.md)
+- **Helper text**: [See form field](./forms/forms-field.md)
+- **Feedback text**: [See form field](./forms/forms-field.md)
+- **Show text as tooltip**: [See form field](./forms/forms-field.md)
+- **Placeholder text**: [See form field](./forms/forms-field.md)
+- **Counter**: [See form field](./forms/forms-field.md)
+- **Resize behavior**: Determines how textareas can be resized (both directions, horizontally, vertically, no resizing)
+- **Columns and width**: Defines inital width by number of columns and/or width
+- **Rows and height**: Defines inital height by number of rows and/or height
```

Comment:
Also here is this an option or should it be handled in the behavior in context. How can I change it, can I change it, and what should be considered from a UX perspective?